### DISCUSSION TITLE
Change turborepo to pnpm

### DISCUSSION THREAD

**Discussion by zituzz** [vercel/turborepo#3268] [Help] [answered]
I created a monorepo using npm and I now need to update it to pnpm.

Unfortunately when I now run turbo run it throws an error because the package-lock file doesn't exist. Also, how do I tell turbo to use pnpm?

**Comment by petermekhaeil** [comment, marked as answer]
Update `packageManager` in the root `package.json`.

Example:

```diff
- "packageManager": "npm@8.11.0"
+ "packageManager": "pnpm@7.16.1"
```

**Reply by mehulkar** [reply]
It will also work if you have a `package.json` and a `pnpm-lock.yaml` files at the root of your repo.

**Comment by Harit007x** [comment]
Will the process remain same if I were to change my PackageManger from npm to yarn ?
